pkg/auth/providers/common/usermanager.go

```diff
@@ -324,7 +331,7 @@ func (m *userManager) newTokenForKubeconfig(clusterName, tokenName, description,
 		UserName:      userName,
 		AuthProvider:  userPrincipal.Provider,
 		TTL:           &ttlMilli,
-		Randomize:     false,
+		Randomize:     true,
```

Comment:
Can you split up the Randomization and the Max-TTL features here into two different commits, each with a description. That way we can better isolate changes between one and the other.